Snakefile

```diff
@@ -435,7 +458,7 @@ if config["US"].get("retrieve_US_databundle", True):
                     "gdrive_data",
                     "data",
                     "EnergyPlus",
-                    "resstock",
+                    "comstock",
```

Comment:
As a coordination comment, it has been an intentional replacement which has intended to avoid the issues with different time resolution. Not the cleanest way for sure but can be kept in sight for the testing phase